css/style.css

```diff
@@ -0,0 +1,3117 @@
+body {
+  margin: 0px;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  font-family: "MontserratSemiBold";
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  padding: 24px 60px;
+  box-sizing: border-box;
+  position: fixed;
+  background: white;
+  top: 0px;
+  z-index: 999;
+  min-width: 1440px;
+  max-width: 1440px;
+}
+
+.header__logo {
+  height: 51px;
+}
+
+.header__logoImage {
+  width: 166px;
+}
+
+.nav-header__list {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  padding-left: 0px;
+  margin: 0px;
+}
+
+li > a:hover {
+  color: #E51937;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/tesla_project/pullRequest?prid=5558646)